examples/abbas_work_area/file_manager.py

```diff
@@ -0,0 +1,106 @@
+import os
+
+CONTACT_FILE_PATH = "C:/Users/71519/Documents/genz-todo-list-app/examples/abbas_work_area/data/store_contacts.csv"
```

Comment:
We should use a relative path here, as absolute path will not let this script work on different hosts